jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/RemoteExceptionMapper.java

```diff
@@ -38,10 +40,23 @@
     @Override
     public Response toResponse(RemoteException exception) {
         String errorId = UUID.randomUUID().toString();
-        log.error("Error handling request {}", SafeArg.of("errorId", errorId), exception);
 
-        SerializableError error = exception.getRemoteException();
         Status status = Status.fromStatusCode(exception.getStatus());
+
+        // here in the client, log responses that indicate client error (4xx) at higher level than server error (5xx)
+        if (CLIENT_ERROR.equals(status.getFamily())) {
+            log.error("Received response status code {} from server handling request. errorId: {}",
+                    SafeArg.of("statusCode", status.getStatusCode()),
+                    SafeArg.of("errorId", errorId),
+                    exception);
+        } else {
+            log.warn("Received response status code {} from server handling request. errorId: {}",
```

Comment:
Without configuration, generally logging libraries log at INFO level and higher (WARN, ERROR, FATAL), and our dropwizard-esque project has those defaults too.